packages/vscode-dependencies-validation/src/util.ts

```diff
@@ -0,0 +1,46 @@
+import type { DiagnosticCollection, Uri } from "vscode";
+import { dirname } from "path";
+import {
+  findDependencyIssues,
+  invokeNPMCommand,
+} from "@sap-devx/npm-dependencies-validation";
+import { isEmpty } from "lodash";
+import { VscodeOutputChannel } from "./vscodeTypes";
+
+const NOT_IN_NODE_MODULES_PATTERN =
+  /^(?!.*[\\|\/]node_modules[\\|\/]).*[\\|\/].+/;
+
+export function isNotInNodeModules(uri: Uri): boolean {
+  return NOT_IN_NODE_MODULES_PATTERN.test(uri.fsPath);
+}
+
+export function clearDiagnostics(
```

Comment:
What value does this one liner function provide?

Reply:
I use it several times in the code